README.md

```diff
@@ -26,6 +27,7 @@ This is a collection of awesome resources related to the yearly
     * [Kotlin](#kotlin)
     * [Nim](#nim)
     * [Perl](#perl)
+    * [PHP](#php)
```

Comment:
`H` technically comes before `e` so please move this above `Perl` (applies to the PHP section itself, too)